website/docs/migration-guides/yew/from-0_18_0-to-0_19_0.mdx

```diff
@@ -125,9 +125,9 @@ html! {
 -   `TimeoutService`
     Use [`gloo-timers`](https://docs.rs/gloo-timers/) instead.
 -   `WebSocketService`
-    Use [`wasm-sockets`](https://github.com/scratchyone/wasm-sockets) or [`reqwasm`](https://github.com/hamza1311/reqwasm) instead.
+    Use [`wasm-sockets`](https://github.com/scratchyone/wasm-sockets) or [`gloo-net`](https://github.com/rustwasm/gloo) instead.
```

Comment:
```suggestion
    Use [`wasm-sockets`](https://github.com/scratchyone/wasm-sockets) or [`gloo-net`](https://crates.io/crates/gloo-net) instead.
```

I'd prefer links to crates.io for links to gloo libraries (also in the other mentions).

Reply:
Sounds good. I was on the fence about this. Updated.